source/guide_forgejo.rst

```diff
@@ -0,0 +1,519 @@
+  command=%(ENV_HOME)s/forgejo/forgejo web
+  startsecs=30
+  autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+.. note:: The status of forgejo must be ``RUNNING``. If its not check the log output at ``~/logs/supervisord.log`` and the configuration file ``~/forgejo/custom/conf/app.ini``.
+
+Uberspace web backend
+---------------------
+
+.. note:: forgejo is running on port 3000.
+
+.. include:: includes/web-backend.rst
+
+Done. We can point our browser to https://isabell.uber.space/.
+
+Installed files and folders are:
+
+* ``~/forgejo``
+* ``~/etc/services.d/forgejo.ini``
+
+Backup
+======
+
+The Forgejo CLI (command line interface) has a build-in backup command to create a full backup in a zip file with the database, repos, config, log, data
+
+.. note:: To restore a backup follow the `backup and restore documentation <https://docs.gitea.io/en-us/backup-and-restore/#restore-command-restore>`_
+
+Execute the following command:
+
+::
+
+  [isabell@stardust ~]$ ~/forgejo/forgejo dump
+  2023/01/16 20:51:33 ...dules/setting/log.go:288:newLogService() [I] Gitea v1.18.0-1 built with GNU Make 4.1, go1.19.4 : bindata, sqlite, sqlite_unlock_notify
+  2023/01/16 20:51:33 ...dules/setting/log.go:335:newLogService() [I] Gitea Log Mode: Console(Console:info)
+  [...]
+  2023/01/16 20:51:34 cmd/dump.go:245:runDump() [I] Dumping local repositories... /home/isabell/forgejo/data/forgejo-repositories
+  2023/01/16 20:51:34 cmd/dump.go:283:runDump() [I] Dumping database...
+  [...]
+  2023/01/16 20:51:34 cmd/dump.go:295:runDump() [I] Adding custom configuration file from /home/isabell/forgejo/custom/conf/app.ini
+  2023/01/16 20:51:34 cmd/dump.go:323:runDump() [I] Packing data directory.../home/isabell/forgejo/data
+  [isabell@stardust ~]$
+
+Updates
+=======
+
+.. note:: Check the update feed_ or releases_ page regularly to stay informed about the newest version.
+
+
+Manual updating
+---------------
+
+* Stop the application ``supervisorctl stop forgejo``
+* Do the *Download* (and optionally *Verifying*) part from above.
+* Check if you have to modify the config file. (See documentation_ and the `file sample <https://codeberg.org/forgejo/forgejo/raw/branch/main/custom/conf/app.example.ini>`_.)
+* Do the database migration: ``~/forgejo/forgejo migrate``
+* Start the application ``supervisorctl start forgejo``
+* Check if the application is running ``supervisorctl status forgejo``
+
+
+Automated updating by custom script
+-----------------------------------
+
+You can also automate the update by using a custom script that automatically executes all the update steps. Create ``~/bin/forgejo-update`` with the following content:
+
+.. code-block:: bash
+
+  #!/usr/bin/env bash
+
+  APP_NAME=Forgejo
+  FORGEJO_LOCATION=$HOME/forgejo/forgejo
+  TMP_LOCATION=$HOME/tmp
+  GPG_KEY_FINGERPRINT=7C9E68152594688862D62AF62D9AE806EC1592E2
+
+  ORG=forgejo # Organisation or GitHub user
+  REPO=forgejo
+  GITHUB_API_URL=https://codeberg.org/$ORG/$REPO/releases/latest
```

Comment:
I suggest to remove the update script for now. Maybe we can add an adjusted version later when this guide is published with the basic manual steps.

Since there is no `latest` tagging system for versions on Forgejo/Gitea like we have on GitHub this does not work.
This latest link redirects to the last non-pre-release, so it could also be a maintenance release for a lower major version. 
You have to get the releases via `https://codeberg.org/api/v1/repos/forgejo/forgejo/releases` (a `latest` endpoint does not exist) and check for the version tags and also the `prerelease: false` state.
This is much more complicated so the `get_latest_version` function has to be changed a lot with check loops etc.